### PR TITLE
Sample changer bricks improvements

### PR DESCRIPTION
**Pull request by MartinSavko**
Qt4_CatsSimpleBrick
 -- allow CatsStatusView to inherit from BlissWidget in addition to QGroupBox
 -- improving log messages
Qt4_SampleChangerBrick3
 -- More flexible layout of samples. Redefining mouse vial click events. Not limiting number of samples per basket to 10.

### CODE REVIEW

File: BlissFramework/Bricks/Qt4_CatsSimpleBrick.py [hunk 1]
```diff
@@ -59,13 +62,13 @@ def __init__(self, parent):
...
     def setStatusMsg(self, status):
-        logging.getLogger("HWR").debug(">>>>> CATS Brick Setting status msg to " + str(status)) 
         self.status_label.setToolTip(status)
         status = status.strip()
         self.setToolTip(status)
 
     def setState(self, state):
-        logging.getLogger("HWR").debug(">>>>> CATS Brick Setting state to " + str(state)) 
+       
+        logging.getLogger("HWR").debug("SC StatusView. State changed %s" % str(state))
```
Comment (IvarsKarpics) [inline BlissFramework/Bricks/Qt4_CatsSimpleBrick.py]: I am not sure if the gui bricks is the place to print log lines with type HWR?

**Review by IvarsKarpics — CHANGES_REQUESTED** [review]

**Comment by MartinSavko** [comment]
Indeed Qt4_CatsMaintBrick, Qt4_CatsSimpleBrick are the only ones that do that. 

sed -i 's/logging.getLogger("HWR")/logging.getLogger()/g' Qt4_CatsMaintBrick.py Qt4_CatsSimpleBrick .py

setting to default ...

**Comment by IvarsKarpics** [comment]
Sorry for counting peanuts, but Qt4_SampleChangerBrick3.py is full with logging lines on each mouse click. Maybe we should introduce even fine grained logging level that could be disabled at the startup.

**Comment by MartinSavko** [comment]
Removing HWR logger from Qt4_SampleChangerBrick3. Indeed flooding log with every vial click is superfluous -- those messages are at the debug level at the moment.

**Pull request merged**